### PR TITLE
PRC-9 : Audit queue integration

### PR DESCRIPTION
**Pull request by prabash-moj**
[PRC-9](https://dsdmoj.atlassian.net/browse/PRC-9) : Audit queue integration

[PRC-9]: https://dsdmoj.atlassian.net/browse/PRC-9?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: helm_deploy/hmpps-contacts-ui/values.yaml
Comment (timharrison-moj) [inline helm_deploy/hmpps-contacts-ui/values.yaml:56]: This will fail to start up on dev without first creating the secret `sqs-hmpps-audit-secret` in the namespace. This is the bit of cloud platform work required (mentioned earlier)

**Review by timharrison-moj — APPROVED** [review]
LGTM. Though please don't merge until we have the k8s secrets, and access to the audit queue from our dev namespace. Have a look at other cloud platform environments - to see how this is done, or give me a shout tomorrow.

**Review by james-baker-moj — APPROVED** [review]
LGTM but take note of Tims comment before merging

**Review by timharrison-moj — COMMENTED** [review]
Could we have a config switch to enable/disable auditing for an environment please? BVLS does this - it means we can turn off and on selectively through an environment var.

File: helm_deploy/hmpps-contacts-ui/values.yaml
Comment (prabash-moj) [reply helm_deploy/hmpps-contacts-ui/values.yaml:56]: raised a [PR](https://github.com/ministryofjustice/cloud-platform-environments/pull/25425) and merged it now, this is ready for merge

**Comment by prabash-moj** [comment]
> Could we have a config switch to enable/disable auditing for an environment please? BVLS does this - it means we can turn off and on selectively through an environment var.

its already available I here isnt it the code required?, https://github.com/ministryofjustice/hmpps-contacts-ui/blob/835cdece7f70e55bee1cf24080319b30cfc7eaa0/server/config.ts#L37

**Pull request merged**
